content/docs/operating/security.md

```diff
@@ -161,3 +161,16 @@ members of the Prometheus development team and the staff of those providers
 have access. If you are concerned about the exact provenance of your binaries,
 it is recommended to build them yourself rather than relying on the
 pre-built binaries provided by the project.
+
+## External audits
+
+[CNCF](https://cncf.io) sponsored an external security audit by
+[cure53](https://cure53.de) which ran from April 2018 to June 2018.
+
+The audit found concerns regarding Prometheus' secrity model, but following
+what is layed out in this document should guard against most of them.
+In particular, the CORS (PRM-01-001) and CSRF (PRM-01-003) attack vectors might
```

Comment:
They're in this doc now, and many attack vectors are non-obvious. Putting it like this make it seem as though reading this doc is not sufficient.